android-sdk/src/main/java/com/checkout/android_sdk/logging/FramesLoggingEventDataProvider.kt

```diff
@@ -27,4 +27,41 @@ object FramesLoggingEventDataProvider {
         )
     }
 
+    fun logThreedsWebviewLoadedEvent(
+        isSuccessFullWebviewLoadingOrCompletionEvents: Boolean,
+    ): FramesLoggingEvent {
+        val eventData = mapOf(
+            WebviewEventAttribute.success to isSuccessFullWebviewLoadingOrCompletionEvents,
+        )
+        return FramesLoggingEvent(
+            if (isSuccessFullWebviewLoadingOrCompletionEvents) MonitoringLevel.INFO else MonitoringLevel.ERROR,
+            FramesLoggingEventType.THREEDS_WEBVIEW_LOADED,
+            eventData
+        )
+    }
+
+    fun logThreedsWebviewCompleteEvent(
+        tokenID: String?,
+        isSuccessFullWebviewLoadingOrCompletionEvents: Boolean,
+    ): FramesLoggingEvent {
+        val tokenIDToPass = tokenID ?: ""
+        val eventData = mapOf(
+            WebviewEventAttribute.tokenID to tokenIDToPass,
+            WebviewEventAttribute.success to isSuccessFullWebviewLoadingOrCompletionEvents,
```

Comment:
these variable names are longer than necessary. Can we just use `tokenID` and `success`?

Reply:
Ok, I will change these variables' names to all the remaining 3ds logging events as well.